src/ui-kit/form-controls/date/date.component.ts

```diff
@@ -490,19 +499,19 @@ export class SamDateComponent
   }
 
   triggerTouch() {
-    this.onTouched();
+    // this.onTouched();
```

Comment:
@yerramshilpa It looks like this is just a wrapper call to call the onTouched callback, which gets called by ControlValueAccessor ideally whenever a control in your component receives focus. It would be better to just not call `triggerTouch` than to comment this out.

Also, better to delete something completely than comment it out.

Reply:
Forgot to undo changes